doc/en/developer/source/quickstart/intellij.rst

```diff
@@ -86,4 +96,7 @@ Run GeoServer from Intellij on Windows
 --------------------------------------
 
 #. Add bash to your Windows environment path and restart Intellij.  
-#. If there are errors such as "cannot find symbol AbstractUserGroupServiceTest", rebuild the security-tests project in the security module.  Right click on the security-tests project and click Rebuild.  
+#. If there are errors such as "cannot find symbol AbstractUserGroupServiceTest", rebuild the security-tests project in the security module.  Right click on the security-tests project and click Rebuild.
```

Comment:
These two are not really Windows specific.

Reply:
Here I mean, they are good comments, but don't belong in the Windows specific section, should be above, in the general introduction.

Reply:
Moved in the latest commit. So this should appear as outdated